src/constants.js

```diff
@@ -17,13 +18,19 @@ export const DECIMAL_PLACES = 2;
 
 /**
  * ID of the genesis block
+ *
+ * @deprecated since version 0.24.0.
+ * Will be deleted in version 1.0.0
```

Comment:
I like the deprecate tag, I just don't think we should set a version to deprecate it.

We can write "Will be deleted in the near future", or something like that. We don't know when we are going to release a v1.0.0

What do you think?

Reply:
The logic behind putting 1.0.0 as the deprecation version is that when we remove those constants, this should be considered a breaking change. So they will be removed in the next major version, which is 1.0.0.

It wouldn't make sense to remove them in some `v0.*`

And it wouldn't also make sense to release a v1.0.0 without removing those constants.

So, by this logic, it will be surely on v1.0.0.

Does it make sense?
But

Reply:
Unless we are currently in a period of not guaranteeing the backwards compatibility of minor versions.
Are we?

In this case, we could remove them in `v0.*`.

But this fact of not guaranteeing backwards compatibility should be stated somewhere in the docs.